docs/DeveloperGuide.md

```diff
@@ -813,48 +813,116 @@ testers are expected to do more *exploratory* testing.
+   Expected: The main window and all minimized windows should close automatically.
 
-    1. Test case: `vcreate n/Little Johnny p/98765432 e/littlejohnny@example.com s/little`<br>
-       Expected: Volunteer is created and added to the bottom of the volunteer list. Details of the volunteer shown in the status message.
+### Creating a volunteer
 
-    1. Test case: `vcreate n/Little Johnny p/91234567 e/bigjohnny@example.com s/little`<br>
-       Expected: Volunteer is not created as there already exists a 'Little Johnny' in the volunteer list. Error details shown in the status message.
+1. Test case: `vcreate n/Little Johnny p/98765432 e/littlejohnny@example.com s/little`<br>
+   Expected: Volunteer is created and added to the bottom of the volunteer list. Details of the volunteer shown in the status message.
 
-    1. Other incorrect delete commands to try: `vcreate`, `vcreate n/`, `...` <br>
-       Expected: `Invalid command format!` error message shown in the status message. Instructions on the correct format are shown as well.
+1. Test case: `vcreate n/Little Johnny p/91234567 e/bigjohnny@example.com s/little`<br>
+   Expected: Volunteer is not created as there already exists a 'Little Johnny' in the volunteer list. Error details shown in the status message.
+
+1. Other incorrect delete commands to try: `vcreate`, `vcreate n/`, `...` <br>
+   Expected: `Invalid command format!` error message shown in the status message. Instructions on the correct format are shown as well.
 
 ### Deleting a volunteer
 
-1. Prerequisites: List all volunteers using the `vlist` command. Multiple volunteers in the list.
+1. Deleting a volunteer while all volunteers are being shown
+
+  1. Prerequisites: List all volunteers using the `vlist` command. Multiple volunteers in the list.
+
+  1. Test case: `vdelete 1`<br>
+     Expected: First volunteer is deleted from the list. Details of the deleted volunteer shown in the result display.
+
+  1. Test case: `vdelete 0`<br>
+     Expected: No volunteer is deleted. Error details for invalid command format is shown in the result display.
+     The outcome is the same whenever the volunteer index parameter is a non-positive integer.
+
+  1. Test case: `vdelete` <br>
+     Expected: Similar to previous.
+
+  1. Test case: `vdelete x` (where x is a positive integer, larger than the list size) <br>
+     Expected: No volunteer is deleted. Error details for invalid volunteer index is shown in the result display.
+
+### Listing all events
+
+1. Listing all events when current event list panel is filtered.
+
+   1. Prerequisites:
+
+      1. The list of events should have at least 1 event. Otherwise, add in some events using the `ecreate` command.
 
-    1. Test case: `vdelete 1`<br>
-       Expected: First volunteer is deleted from the list. Details of the deleted volunteer shown in the status message.
+      2. Perform `efind n/abc` to get a filtered event list panel (can consist of 0 or more events).
```

Comment:
i think this line can be smth like "Perform an efind command to get a filtered event list panel..." cos `n/abc` is abit random?

Reply:
but ok ill give that as an example instead